scipy/optimize/optimize.py

```diff
@@ -2397,20 +2399,87 @@ def bracket(func, xa=0.0, xb=1.0, args=(), grow_limit=110.0, maxiter=1000):
     return xa, xb, xc, fa, fb, fc, funcalls
 
 
-def _linesearch_powell(func, p, xi, tol=1e-3):
+def _m_min(a, b):
```

Comment:
These can be removed now, right?

Reply:
Ah yes sorry about that, thanks for pointing that out. They are removed.